front/src/Main.elm

```diff
@@ -41,7 +41,7 @@ type alias Model =
     , availableMaterials : MaterialSet
     , units : Units
     , mode : Mode
-    , substituteWhiskey : Bool
+    , pedantic : Bool
```

Comment:
Added a new pedantic mode, on by default, that does several substitutions (see below).